src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -25,33 +36,52 @@ export default {
     target_dataset_tooltip: T(''),
 
     recursive_placeholder: T('Recursive'),
-    recursive_tooltip: T(''),
+    recursive_tooltip: T('Replicate all child dataset snapshots.'),
 
     custom_snapshots_placeholder: T('Replicate Custom Snapshots'),
-    custom_snapshots_tooltip: T(''),
+    custom_snapshots_tooltip: T('Replicate snapshots that have not been \
+ created by an automated snapshot task. Requires defining the naming \
+ schema for the custom snapshots.'),
 
     naming_schema_placeholder: T('Naming Schema'),
-    naming_schema_tooltip: T(''),
+    naming_schema_tooltip: T('Custom snapshots to be replicated. Enter \
+ the name and \
+ <a href="https://www.freebsd.org/cgi/man.cgi?query=strftime" target="_blank">strftime(3)</a> \
+ <i>%Y</i>, <i>%m</i>, <i>%d</i>, <i>%H</i>, and <i>%M</i> strings that \
+ match the snapshots to include in the replication. The number of \
+ matching snapshots is shown.'),
 
     encryption_placeholder: T('SSH Transfer Security'),
-    encryption_tooltip: T('Data transfer security. The connection is authenticated with SSH,\
- then data can either be encrypted during transfer to maximise protection or left unencrypted\
- to maximise transfer speed. Transferring data without encryption is only recommended for\
- secure networks.'),
+    encryption_tooltip: T('Data transfer security. The connection is \
+ authenticated with SSH, then data can either be encrypted during \
+ transfer to maximise protection or left unencrypted to maximise \
+ transfer speed. Transferring data without encryption is only \
```

Comment:
Flip the recommendation: Encryption is recommended, but can be disabled for increased speed on secure networks.